superchain/configs/mainnet/automata.toml

```diff
@@ -1,27 +1,30 @@
 name = "Automata Mainnet"
-chain_id = 65536
 public_rpc = "https://rpc.ata.network"
 sequencer_rpc = "https://automata-mainnet.alt.technology/"
 explorer = "https://explorer.ata.network"
 superchain_level = 0
+governed_by_optimism = false
+data_availability_type = "alt-da"
+chain_id = 65536
 batch_inbox_addr = "0xff00000000000000000000000000000001111111"
```

Comment:
Could be saved for future work but while we're in here -- we could moved batch inbox address and gas paying token to `addresses` table? Or the `addresses` table gets renamed to something more specific such as `l1_contract_addresses`.

Reply:
see above comment re: the addresses struct.